decidim-conferences/app/commands/decidim/conferences/admin/update_partner.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Conferences
+    module Admin
+      # A command with all the business logic when updating a conference
+      # speaker in the system.
+      class UpdatePartner < Rectify::Command
+        # Public: Initializes the command.
+        #
+        # form - A form object with the params.
+        # conference_partner - The ConferencePartner to update
+        def initialize(form, conference_partner)
+          @form = form
+          @conference_partner = conference_partner
```

Comment:
Same as the creation form, if I'd move this inside the form so the command doesn't need to check anything else.